demo/website/src/components/ProjectsList.js

```diff
@@ -23,29 +16,36 @@ class ProjectsList extends Component {
   }
 
   componentDidMount() {
+    const { elasticsearch } = this.props;
```

Comment:
I think the best place to get the dependency is in the top of the file as usual.
The client instantiation would make sense to be in the constructor function, no?
I don't think the client really needs to know anything about the DOM?